src/plugins/OauthPlugin/OauthPlugin.js

```diff
@@ -50,7 +72,7 @@ export default class OauthPlugin extends PluginBase {
       oauthPopup: options.oauthPopup,
     });
 
-    this.frameStrategy.on(FrameStrategy.EVENT_UPDATE_TARGET, target => {
+    this.frameStrategy.on(FrameStrategy.EVENT_UPDATE_TARGET, /** @param {string} target */ target => {
```

Comment:
`target` not a string, target is `contentWindow`(in iframe) or just `window` (in popup mode)
btw it can be `{}` when frame is dropped
or `null` when frame is inited

Reply:
@dudiq look at `@endpass/class/CrossWindowMessenger.d.ts` declaration. It can be `Window` or `{}` only. Maybe declaration in the class incorrect. If so, it should be fixed separately. For now take signature from `CrossWindowMessenger` class.